blue_hardware_drivers/src/BLDCControllerClient.cpp

```diff
@@ -615,6 +615,14 @@ void BLDCControllerClient::queueSetCommand(comm_id_t board_id, float value) {
   queuePacket(board_id, packet);
 }
 
+void BLDCControllerClient::queueSetPosCommand(comm_id_t board_id, float position, float feed_forward) {
+  float values[2];
+  values[0] = position;
+  values[1] = feed_forward;
```

Comment:
Nit: `float values[2] = {position, feed_forward};` is more compact? Also consider renaming to `feedforward`; that's how it's spelled in the hardware interface